components/bundles/BundlesOverview.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect, useCallback } from "react"
+import { useRouter } from "next/router"
+import styles from "./css/BundleOverview.module.css"
+import { BundleRow } from "./BundleRow"
+import { useBundleData } from "../../context/BundleData/BundleDataProvider"
+import BundleModal from "./BundleModal"
+import clsx from "clsx"
+
+export default function BundlesOverview() {
```

Comment:
please check the rendering conditions of this component since it's constantly rerendering itself

simple console log example:
![image](https://user-images.githubusercontent.com/1703831/123530608-c3b7a200-d6d2-11eb-8a20-60413fc754ee.png)